model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/velocity/velocity_advection.py

```diff
@@ -112,54 +113,40 @@ def run_predictor_step(
         cell_areas: fa.CellField[float],
     ):
         cfl_w_limit, scalfac_exdiff = self._scale_factors_by_dtime(dtime)
-
-        start_vertex_lb_plus1 = self.grid.get_start_index(
-            dims.VertexDim, h_grid.HorizontalMarkerIndex.lateral_boundary(dims.VertexDim) + 1
-        )
-        end_vertex_local_minus1 = self.grid.get_end_index(
-            dims.VertexDim, h_grid.HorizontalMarkerIndex.local(dims.VertexDim) - 1
+        vertex_domain = h_grid.domain(dims.VertexDim)
+        edge_domain = h_grid.domain(dims.EdgeDim)
+        cell_domain = h_grid.domain(dims.CellDim)
+        start_vertex_lateral_boundary_row_2 = self.grid.start_index(
```

Comment:
More consistency renaming
```suggestion
        start_vertex_lateral_boundary_level_2 = self.grid.start_index(
```